capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -1 +1,19 @@
 package com.google.univiz.api;
+
+/*
+ * The MapData interface is responsible for all data management/processing specific for the maps
+ * feature of the UniViz webapp.
+ */
+public interface MapData {
+  /*
+   * Takes a list of college names and returns a CollegeMapsData object to the caller. This object
+   * contains all data relevant to the Maps feature.
+   *
+   * @args Takes a list of CollegeIDs, which is a class with two components: a college name and a 
```

Comment:
I don't think `@args` is valid Javadoc. I think you want `@param`. Also, if you `@link` to the type involved you won't have to explain the type in this documentation. The advantage of that is that if the content of that type changes you won't have to update this comment.